pkg/reconciler/certificate/config/cert_manager.go

```diff
@@ -51,9 +51,9 @@ type CertManagerConfig struct {
 func NewCertManagerConfigFromConfigMap(configMap *corev1.ConfigMap) (*CertManagerConfig, error) {
 	// Use Knative self-signed ClusterIssuer as default
 	config := &CertManagerConfig{
-		IssuerRef:               knativeSelfSignedIssuer,
-		ClusterLocalIssuerRef:   knativeSelfSignedIssuer,
-		SystemInternalIssuerRef: knativeSelfSignedIssuer,
+		IssuerRef:               knativeSelfSignedIssuer.DeepCopy(),
+		ClusterLocalIssuerRef:   knativeSelfSignedIssuer.DeepCopy(),
+		SystemInternalIssuerRef: knativeSelfSignedIssuer.DeepCopy(),
```

Comment:
Unmarshalling was affecting all pointers btw. So this returns a new pointer for each case with the same content.